Backend/KFC_WebAPI/Controllers/ApplicationController.cs

```diff
@@ -218,15 +201,14 @@ public IHttpActionResult GenerateKey([FromBody] ApplicationRequest request)
 
             try
             {
-                using (var _db = new DatabaseContext())
```

Comment:
Why is "using" being removed?

Reply:
I declared it once at the top so there's no duplicate code because I noticed all of the methods were creating a new DatabaseContext each time

Reply:
if using is not used, then the context is not being disposed, ef6 database context disposable

Reply:
Ok I see... I will fix them and do another pull request... i also updated the health check route to work "concurrently" so that will be part of the updated pull request.

Reply:
I updated and added a bunch of stuff (sorry in advance)...

Reply:
There was a merge conflict that I resolved.